spec/helpers/table_helper_spec.rb

```diff
@@ -0,0 +1,82 @@
+require "rails_helper"
+
+RSpec.describe TableHelper, type: :helper do
+  describe "#sort_table_header", :sort_table_header do
+    let(:organisation) { build(:organisation) }
+    let(:header_label) { "Last Updated" }
+    let(:header_content_attribute) { "public_updated_at" }
+    let(:path_params) {
+      {
+        organisation_id: organisation
+      }
+    }
+
+    before do
+      assign(:organisation, organisation)
+    end
+
+    describe 'Accessibility' do
+      context 'When the column is unsorted' do
+        it "sets aria-sort to none " do
+          header = helper.sort_table_header(header_label, header_content_attribute, nil, nil)
```

Comment:
The two extra parameters (`nil`, `nil`) would go away if we don't pass them as parameters and use `params` directly.